editor/src/clj/editor/code/script.clj

```diff
@@ -506,78 +511,77 @@
       (pair proj-path line-number))))
 
 (g/defnk produce-build-targets [_node-id resource lines lua-preprocessors script-properties module-build-targets original-resource-property-build-targets]
-  (if-some [errors
```

Comment:
This is mostly indentation, due to wrapping everything in a `(let [workspace (resource/workspace resource)] ...)` expression. We need the workspace to look up the supported file extensions for the atlas resource type.